test-sample-test.ts

```
@@ -0,0 +1,148 @@
+}
+
+// Issue 5: No input validation
+function divide(a: number, b: number): number {
+  return a / b; // Division by zero not handled
+}
+
+// Issue 6: Memory leak potential
+class DataProcessor {
+  private data: any[] = [];
+  private timers: NodeJS.Timeout[] = [];
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)